### DISCUSSION TITLE
Permitted processes: MsWord & MsExcel

### DISCUSSION THREAD

**Discussion by Rohanlochen** [SafeExamBrowser/seb-mac#281] [Q&A] [answered]
Hey all, my team and I are currently struggling with allowing permitted processes such as Word and Excel. Uploaded are our configuration settings and we are at a roadblock regarding this issue. Any and all help is much appreciated - We are trying to implement this and roll SEB out once it passes its Proof Of Concept phase. No issues with the windows side of things. Is there an apparent solution within our config file that we haven't noticed?
[permitted applications [msword + msexcel].zip](https://github.com/SafeExamBrowser/seb-mac/files/10917497/permitted.applications.msword.%2B.msexcel.zip)

**Comment by danschlet** [comment, marked as answer]
Running third party applications despite the permitted processes setting already being available is **currently not possible in the macOS version**, as the manual states.

We will add such a feature later this year, although because it will be based on Apple's API in the macOS Assessment Mode (AAC kiosk mode, not the default SEB kiosk mode), it's unclear if this will work also with complex applications like Word and Excel. Apple intends this feature to used with simple tools for example for accessibility purposes. **Also the feature doesn't stop applications from ACCESSING THE FILE SYSTEM!** So don't expect this to allow perfectly secure exams.

See this discussion threads:
https://github.com/SafeExamBrowser/seb-mac/discussions/114
https://github.com/SafeExamBrowser/seb-mac/discussions/276

**Reply by Rohanlochen** [reply]
Thank you for your very swift reply Mr Schneider,

Its extremely appreciated & well received by my team and I. We look forward to your team adding this feature down the road and we are keen to test it and implement it when its out! We will move on now with our proof of concept testing without app filtering for macOS now that we have a definite answer and hopefully get SEB rolled out within the next couple of intakes for other BYOD users.

Cheers all the way from Australia.

Rgds - Rohan
